content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md

```diff
@@ -15,97 +15,143 @@ tags:
 
-   ![InstallNetDAppRunningOptionsSCALE](/images/SCALE/Apps/InstallNetDAppRunningOptionsSCALE.png "Netdata App Active")
+You can add extra storage volumes at the time of installation or edit the application after it deploys. Stop the app before editing settings.
 
-## Using the Netdata Web Portal
+### Advanced DNS Settings
+The default **DNS Configuration** are sufficient for a basic installation. 
+To specify additional DNS options, click **Add** to the right of **DNS Options** to add the DNS **Option Name** and **Option Value** fields.
+
+{{< trueimage src="/images/SCALE/Apps/InstallNetdataAdvancedDNSOptions.png" alt="Install Netdata Advanced DNS Options" id="Install Netdata Advanced DNS Options" >}}
 
-A successfully installed Netdata app displays in the **Installed Applications** tab with a status of **Active**.
+### Resource Limits Settings
+Accept the default values in **Resources Limits** or select **Enable Pod Resource limits** to show CPU and memory resource configuration options.
 
-   ![InstallNetDAppRunningSCALE](/images/SCALE/Apps/InstallNetDAppRunningSCALE.png "Netdata App Installed")
+By default, the application is limited to use no more than four CPU cores and eight gigabytes available memory. 
+The application might use considerably less system resources.
 
-1. Click on the **Web Portal** button. The Netdata agent dashboard displays. The Netdata agent dashboard provides a system overview that displays CPU usage and other vital statistics.
+{{< trueimage src="/images/SCALE/Apps/InstallNetdataResourceLimitsEnablePod.png" alt="Install Netdata Resource Limits" id="Install Netdata Resource Limits" >}}
 
-   ![InstallNetDAppNetDAgentCropSCALE](/images/SCALE/Apps/InstallNetDAppNetDAgentCropSCALE.png "Netdata Agent Dashboard")
+To customize the CPU and memory allocated to the container (pod) Netdata uses, enter new CPU values as a plain integer value followed by the suffix m (milli).
+Default is 4000m.
+
+Accept the default value 8Gi allocated memory or enter a new limit in bytes.
+Enter a plain integer followed by the measurement suffix, for example 129M or 123Mi.
+
+## Using the Netdata Web Portal
+After deploying the SCALE Netdata app click on **Web Portal** to open the Netdata agent local dashboard. 
+This dashboard provides a system overview with CPU usage and other vital statistics.
 
-2. The Netdata agent displays a limited portion of the reporting capabilities of the Netdata app. Click on the *Node View* tab to better understand the differences between the Netdata agent and Netdata Cloud. Evaluate your system reporting needs.
+{{< trueimage src="/images/SCALE/Apps/NetdataWebPortalLocalDashboard.png" alt="Netdata Web Portal Local Dashboard" id="Netdata Web Portal Local Dashboard" >}}
 
-   ![InstallNetDAppNetDAgentDashNoCloudSCALE](/images/SCALE/Apps/InstallNetDAppNetDAgentDashNoCloudSCALE.png "Netdata Agent Node View")
+The Netdata dashboard displays a limited portion of the reporting capabilities of the Netdata app. 
```

Comment:
Not quite sure what you're saying here. Is it that the Netdata dashboard (accessed through SCALE Netdata app) is limited compared to the Netdata Cloud? There are four terms in 147-148 (Netdata dashboard, Netdata app--which may or may not be the SCALE app?--, Netdata agent, and Netdata Cloud) and I'm not sure which is referring to what, if there is an overlap between the first two terms and the second two, etc.

Reply:
This entire section is talking about the Netdata dashboard accessed through their web portal as indicated by the level 2 header. I've added additional clarifying text.